examples/jsm/loaders/GLTFLoader.js

```diff
@@ -1487,7 +1487,7 @@ class GLTFDracoMeshCompressionExtension {
 				const accessorDef = json.accessors[ primitive.attributes[ attributeName ] ];
 				const componentType = WEBGL_COMPONENT_TYPES[ accessorDef.componentType ];
 
-				attributeTypeMap[ threeAttributeName ] = componentType;
+				attributeTypeMap[ threeAttributeName ] = componentType.name;
```

Comment:
I'm confused. Why is this change now necessary?

Reply:
Because I have removed the code from the DRACOLoader which auto-converts types in `attributeTypeMap` from constructors like `Float32Array` to strings like `'Float32Array'`.